### PR TITLE
Latin bows and Spaniard staves

### PR DESCRIPTION
**Pull request by nateo221**
Added Spanish descriptions for bows and wands.

### CODE REVIEW

**Comment by nateo221** [comment]
I'm going to cancel the request and try again

**Comment by nateo221** [comment]
basically just removed my Spanish bow translations in favor of the other guys Spanish

**Comment by nateo221** [comment]
added swords (espadas)

**Comment by dekdevy** [comment]
I checked some of the translations and i really liked your editorials, nice job! when the conflicts are resolved, I can merge. Thank you!

**Comment by nateo221** [comment]
i saw that in the sword tier after the demonsword, most the descriptions are duplicates off the demonsword

**Comment by nateo221** [comment]
the parser travis deal crashed, closing and restarting

**Comment by nateo221** [comment]
all fixed

File: loc/items/sword.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ export default [
     description: {
       en: 'It can cut demons, but not angels',
       ru: 'Oно может резать демонов, но не ангелов',
-      es: 'Puede cortar demonios, pero no ángeles.',
+      es: 'San Dek lo usó en las guerras en los cielos.',
```
Comment (dekdevy) [inline loc/items/sword.js]: Please dont diverge from the other texts to much, "Dek" is not part of the lore :) If you have questions you best ask dhwty in discord about these kind of things.
Other than that I think its great and good to go.

**Comment by nateo221** [comment]
removed mention of the developers

File: loc/items/sword.js (hunk 1 above)
Comment (nateo221) [reply loc/items/sword.js]: fixed the problem

**Comment by dekdevy** [comment]
Thank you a lot for the translations!

**Pull request merged**